packages/tpetra/core/test/CrsGraph/CrsGraph_UnitTests_Swap.cpp

```diff
@@ -125,55 +125,58 @@ TEUCHOS_STATIC_SETUP()
 // gbl_num_columns: Max # of columns in the matrix-representation of the graph.
 //                  This should be >= the highest value of v from all edges (u,v) in edges.
 //                  Note: u and v are 0-indexed, so if the highest v is 11, then this should be 12.
-template<class LO, class GO, class Node, class comm_t>
+template<class LO, class GO, class Node, class comm_type>
 Teuchos::RCP<Tpetra::CrsGraph<LO, GO, Node>>
-generate_crsgraph(Teuchos::RCP<comm_t>                      comm,
+generate_crsgraph(Teuchos::RCP<comm_type>                      comm,
                   const std::vector<std::pair<GO, GO>>&     gbl_edges,
                   const std::vector<std::pair<GO, size_t>>& gbl_row_owners,
                   const size_t                              gbl_num_columns,
                   const bool                                do_fillComplete=true)
 {
     using Teuchos::Comm;
 
-    typedef Tpetra::CrsGraph<LO, GO, Node> graph_t;
-    typedef Tpetra::Map<LO, GO, Node>      map_t;
+    typedef Tpetra::CrsGraph<LO, GO, Node> graph_type;
+    typedef Tpetra::Map<LO, GO, Node>      map_type;
 
-    typedef typename std::map<GO, size_t> map_rows_t;      // row_id, num_entries
+    typedef typename std::map<GO, size_t> map_rows_type;      // row_id, num_entries
 
 
-    typedef typename std::vector<GO>        vec_go_t;
-    typedef typename std::map<GO, vec_go_t> map_row_to_cols_t;
+    typedef typename std::vector<GO>        vec_go_type;
+    typedef typename std::map<GO, vec_go_type> map_row_to_cols_type;
+
+    const bool verbose = Tpetra::Details::Behavior::verbose();
 
     const size_t comm_rank = comm->getRank();
```

Comment:
MPI process ranks are always `int`.  This could cause a signed/unsigned conversion warning.

Reply:
@mhoemmen I changed it up so that I can use `int` for `comm_rank`